### PR TITLE
Add send broadcast and start flow bulk actions to contact group page

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/contacts/views.py [hunk 1]
```diff
@@ -990,7 +990,10 @@ def build_content_menu(self, menu):
                 )
 
         def get_bulk_actions(self):
-            return ("block", "archive") if self.group.is_smart else ("block", "unlabel")
+            bulk_actions = ("block", "archive") if self.group.is_smart else ("block", "unlabel")
+            if self.has_org_perm("contacts.contact_update"):
+                bulk_actions += ("send", "start-flow")
```
Comment (rowanseymour) [inline temba/contacts/views.py:995]: shouldn't this be `flows.flow_start` and `msgs.broadcast_create` ?

**Review by rowanseymour — CHANGES_REQUESTED** [review]

File: temba/contacts/views.py (hunk 1 above)
Comment (norkans7) [reply temba/contacts/views.py:995]: I have seen those used in https://github.com/nyaruka/rapidpro/blob/e23a6514791f2b61ef16fe89a601cc5e36987bea/templates/includes/short_pagination.html#L63

And I used `contacts.contact_update` to be consistent with what is used on the `List` view https://github.com/nyaruka/rapidpro/blob/e23a6514791f2b61ef16fe89a601cc5e36987bea/temba/contacts/views.py#L874

Comment (norkans7) [reply temba/contacts/views.py]: Adjust that in both places

File: temba/contacts/views.py [hunk 2]
```diff
@@ -990,7 +995,12 @@ def build_content_menu(self, menu):
                 )
 
         def get_bulk_actions(self):
-            return ("block", "archive") if self.group.is_smart else ("block", "unlabel")
+            actions = ("block", "archive") if self.group.is_smart else ("block", "unlabel")
```
Comment (rowanseymour) [inline temba/contacts/views.py:998]: these also need gated by `contacts.contact_update`

The current `get_bulk_actions` way of working isn't ideal but let's get this fixed this way for now and then we can try to clean it up later.

**Pull request merged**